app/src/main/java/com/x8bit/bitwarden/ui/platform/components/scaffold/BitwardenScaffold.kt

```diff
@@ -52,37 +59,39 @@ fun BitwardenScaffold(
         topBar = topBar,
         bottomBar = bottomBar,
         snackbarHost = snackbarHost,
-        floatingActionButton = {
-            Box(modifier = Modifier.navigationBarsPadding()) {
-                floatingActionButton()
-            }
-        },
+        floatingActionButton = floatingActionButton,
         floatingActionButtonPosition = floatingActionButtonPosition,
         containerColor = containerColor,
         contentColor = contentColor,
-        contentWindowInsets = contentWindowInsets,
+        contentWindowInsets = WindowInsets(0.dp),
```

Comment:
The insets here are cleared and I apply them myself in the content below. This is needed because of the utility bar and the overlay need to go fully edge to edge but the rest of the content should be inset